### PR TITLE
[AUR-176] QGenc and Simulink tests

### PR DESCRIPTION
**Pull request by Ficzer**

### CODE REVIEW

File: requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.asn [hunk 1]
```diff
@@ -0,0 +1,42 @@
+EMPTYPROJECT-DATAVIEW DEFINITIONS ::=
+BEGIN
+
+   MyInteger ::= INTEGER (0 .. 10000)
+
+-- ASN.1 Type definitions must start with an uppercase
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.asn]: Remove commented out code

File: requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.msc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+mscdocument emptyProject /* MSC AND */;
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.msc]: Remove this

File: requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.asn [hunk 3]
```diff
@@ -0,0 +1,42 @@
+INTEGERPARAM-DATAVIEW DEFINITIONS ::=
+BEGIN
+
+   MyInteger ::= INTEGER (0 .. 10000)
+
+-- ASN.1 Type definitions must start with an uppercase
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.asn]: Remove commented out code

File: requirements/AUR_CBI_INT-0170/sequence-param-test/sequenceparam.acn [hunk 4]
```diff
@@ -0,0 +1,12 @@
+SEQUENCEPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0170/sequence-param-test/sequenceparam.acn]: Remove commented out code

File: requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.asn [hunk 5]
```diff
@@ -0,0 +1,43 @@
+IA5STRINGPARAM-DATAVIEW DEFINITIONS ::=
+BEGIN
+
+   MyInteger ::= INTEGER (0 .. 10000)
+   MyIA5String ::= IA5String (SIZE(1..100))
+
+-- ASN.1 Type definitions must start with an uppercase
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.asn]: Remove commented out code

File: requirements/AUR_CBI_INT-0190/partial-vector-update-test/work/system.asn [hunk 6]
```diff
@@ -0,0 +1,17 @@
+-- Dataview generated on-the-fly providing information on the system
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0190/partial-vector-update-test/work/system.asn]: Is this needed?

File: requirements/AUR_CBI_INT-0200/partial-vector-update-test/work/system.asn [hunk 7]
```diff
@@ -0,0 +1,17 @@
+-- Dataview generated on-the-fly providing information on the system
+-- and made available to the user code.
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0200/partial-vector-update-test/work/system.asn:2]: Is this needed?

File: requirements/AUR_CBI_INT-0210/partial-vector-update-test/work/system.asn [hunk 8]
```diff
@@ -0,0 +1,17 @@
+-- Dataview generated on-the-fly providing information on the system
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0210/partial-vector-update-test/work/system.asn]: Is this needed?

File: requirements/AUR_CBI_INT-0220/booleanparam/booleanparam.acn [hunk 9]
```diff
@@ -0,0 +1,12 @@
+BOOLEANPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0220/booleanparam/booleanparam.acn]: Remove commented out code

File: requirements/AUR_CBI_INT-0230/booleanparam/booleanparam.acn [hunk 10]
```diff
@@ -0,0 +1,12 @@
+BOOLEANPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0230/booleanparam/booleanparam.acn]: Remove commented out code

File: requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.acn [hunk 11]
```diff
@@ -0,0 +1,12 @@
+EMPTYPROJECT-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.acn]: Remove the commented out code

File: requirements/AUR_CBI_INT-0065/emptyProject/work/system.asn [hunk 12]
```diff
@@ -0,0 +1,9 @@
+-- Dummy dataview to make the PID type visible at system level
+--  DO NOT EDIT - this file will be overwritten during the build
+System-Dataview DEFINITIONS ::=
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0065/emptyProject/work/system.asn]: Is this needed?

File: requirements/AUR_CBI_INT-0130/real-param-test/realparam.acn [hunk 13]
```diff
@@ -0,0 +1,12 @@
+REALPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0130/real-param-test/realparam.acn]: Remove commented out code

File: requirements/AUR_CBI_INT-0130/real-param-test/work/qgenc/QGenC/src/.qgenc_script.m.md5 [hunk 14]
```diff
@@ -0,0 +1 @@
+b4d054d7adad0ef33c904cef4ab8c64d  src/qgenc_script.m
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0130/real-param-test/work/qgenc/QGenC/src/.qgenc_script.m.md5]: Please confirm - is this needed to avoide the regeneration of the simulink model?

File: requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.acn [hunk 15]
```diff
@@ -0,0 +1,12 @@
+INTEGERPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.acn]: Remove commented out code

File: requirements/AUR_CBI_INT-0140/integer-param-test/work/controller/C/src/controller.c [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+#include "controller.h"
+#include <stdio.h>
+#include <assert.h>
+
+
+void controller_startup(void)
+{
+   // Write your initialisation code, but DO NOT CALL REQUIRED INTERFACES
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0140/integer-param-test/work/controller/C/src/controller.c]: Remove the commented out code

File: requirements/AUR_CBI_INT-0150/boolean-param-test/booleanparam.acn [hunk 17]
```diff
@@ -0,0 +1,12 @@
+BOOLEANPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0150/boolean-param-test/booleanparam.acn]: Remove the commented out code

File: requirements/AUR_CBI_INT-0150/boolean-param-test/work/controller/C/src/controller.c [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+ * Timers              : 
+ */
+
+#include "controller.h"
+#include <stdio.h>
+#include <assert.h>
+
+void controller_startup(void)
+{
+   // Write your initialisation code, but DO NOT CALL REQUIRED INTERFACES
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0150/boolean-param-test/work/controller/C/src/controller.c]: Remove the commented out code

File: requirements/AUR_CBI_INT-0160/sequence-of-data-param-test/sequenceofparam.acn [hunk 19]
```diff
@@ -0,0 +1,12 @@
+SEQUENCEOFPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0160/sequence-of-data-param-test/sequenceofparam.acn]: Remove commented out code

File: requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.acn [hunk 20]
```diff
@@ -0,0 +1,12 @@
+IA5STRINGPARAM-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.acn]: Remove the commented out code

File: requirements/AUR_CBI_INT-0230/booleanparam.md [hunk 21]
```diff
@@ -0,0 +1,19 @@
+# Testing function with boolean type param
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_INT-0230/booleanparam.md:1]: Given that there is no md5 file and other QGenC related producs, isn't the slx model regenerated?

File: requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.asn (hunk 1 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.asn]: Done

File: requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.msc (hunk 2 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.msc]: Done

File: requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.asn (hunk 3 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.asn]: Done

File: requirements/AUR_CBI_INT-0170/sequence-param-test/sequenceparam.acn (hunk 4 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0170/sequence-param-test/sequenceparam.acn]: Done

File: requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.asn (hunk 5 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.asn]: Done

File: requirements/AUR_CBI_INT-0190/partial-vector-update-test/work/system.asn (hunk 6 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0190/partial-vector-update-test/work/system.asn]: Removed

File: requirements/AUR_CBI_INT-0200/partial-vector-update-test/work/system.asn (hunk 7 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0200/partial-vector-update-test/work/system.asn:2]: Removed

File: requirements/AUR_CBI_INT-0210/partial-vector-update-test/work/system.asn (hunk 8 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0210/partial-vector-update-test/work/system.asn]: Removed

File: requirements/AUR_CBI_INT-0220/booleanparam/booleanparam.acn (hunk 9 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0220/booleanparam/booleanparam.acn]: Done

File: requirements/AUR_CBI_INT-0230/booleanparam/booleanparam.acn (hunk 10 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0230/booleanparam/booleanparam.acn]: Done

File: requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.acn (hunk 11 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0065/emptyProject/emptyProject.acn]: Done

File: requirements/AUR_CBI_INT-0065/emptyProject/work/system.asn (hunk 12 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0065/emptyProject/work/system.asn]: Removed

File: requirements/AUR_CBI_INT-0130/real-param-test/realparam.acn (hunk 13 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0130/real-param-test/realparam.acn]: Done

File: requirements/AUR_CBI_INT-0130/real-param-test/work/qgenc/QGenC/src/.qgenc_script.m.md5 (hunk 14 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0130/real-param-test/work/qgenc/QGenC/src/.qgenc_script.m.md5]: Script is not needed. Removed

File: requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.acn (hunk 15 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0140/integer-param-test/integerparam.acn]: Done

File: requirements/AUR_CBI_INT-0140/integer-param-test/work/controller/C/src/controller.c (hunk 16 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0140/integer-param-test/work/controller/C/src/controller.c]: Done

File: requirements/AUR_CBI_INT-0150/boolean-param-test/booleanparam.acn (hunk 17 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0150/boolean-param-test/booleanparam.acn]: Done

File: requirements/AUR_CBI_INT-0150/boolean-param-test/work/controller/C/src/controller.c (hunk 18 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0150/boolean-param-test/work/controller/C/src/controller.c]: Done

File: requirements/AUR_CBI_INT-0160/sequence-of-data-param-test/sequenceofparam.acn (hunk 19 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0160/sequence-of-data-param-test/sequenceofparam.acn]: Done

File: requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.acn (hunk 20 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0180/ia5string-param-test/ia5stringparam.acn]: Done

File: requirements/AUR_CBI_INT-0230/booleanparam.md (hunk 21 above)
Comment (Ficzer) [reply requirements/AUR_CBI_INT-0230/booleanparam.md:1]: No because simulink model is present, see previous projects

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**